content/topics/regular-expressions/_index.md

```diff
@@ -21,7 +21,8 @@ It's not worth memorizing this stuff, but you should understand it.
 
 ## Python Regular expressions
 
-- https://www.w3schools.com/python/python_regex.asp
+- https://www.w3schools.com/python/python_regex.asp - Introduction to Regular Expression.
```

Comment:
We already have a topic on regular expressions. Link there and update it if it needs updating.

Always check if something exists before creating it.